src/common/config.cc

```diff
@@ -113,6 +114,27 @@ int ceph_resolve_file_search(const std::string& filename_list,
   return ret;
 }
 
+int parse_rbd_default_features(const std::string& str_value, uint64_t& value) 
+{
+  int ret = 0;
+  value = 0;
+  try {
+    std::stoi(str_value, NULL, 10);
+  } catch (...) {
+    map<std::string, int> conf_vals = {{"layering", 1}, {"stripingv2", 2}, {"exclusive lock", 4},
+                                       {"object map", 8}, {"fast-diff", 16}, {"deep-flatten", 32}, {"journaling", 64}};
+    std::vector<std::string> strs;
+    boost::split(strs, str_value, boost::is_any_of(","));
+    for (std::vector<std::string>::const_iterator i = strs.begin(); i != strs.end(); ++i)
+      if (conf_vals.find(*i) != conf_vals.end())
+        value += conf_vals[*i];
+      else
+        ret = -1;
```

Comment:
Can you create a new ```RBD_FEATURES_DEFAULT``` constant in src/include/rbd/features.h and set the value to that constant upon failure.